addons/website_slides_survey/views/website_profile.xml

```diff
@@ -9,6 +9,58 @@
                 </div>
             </t>
         </xpath>
+        <!-- Certification Attempts -->
+        <xpath expr="//ul[hasclass('o_wprofile_nav_tabs')]" position="inside">
+            <li class="nav-item">
+                <a role="tab" aria-controls="certification" href="#profile_tab_content_certification" class="nav-link" data-toggle="tab">
+                    <t t-if="certifications" t-esc="len(certifications)" /> Certification Attempts
+                </a>
+            </li>
+        </xpath>
+        <xpath expr="//div[hasclass('o_wprofile_tabs_content')]" position="inside">
+            <div role="tabpanel" class="tab-pane" id="profile_tab_content_certification">
+                <h5 class="d-flex justify-content-between align-items-end border-bottom pb-1">
+                    Certifications Attempts
+                    <t t-call="website.website_search_box_input">
+                        <t t-set="action" t-valuef="/profile/user/#{user.id}"/>
+                        <t t-set="placeholder">Search Attempts...</t>
+                        <t t-set="search" t-value="certification_search_terms or ''"/>
+                        <t t-set="search_key" t-valuef="certification"/>
+                        <t t-set="order_by" t-valuef="create_date"/>
+                    </t>
+                </h5>
+                <t t-if="not certifications">
+                    <p>You have not taken any certification yet.</p>
+                    <a href="/slides/all?slide_category=certification">
+                        <i class="fa fa-arrow-right"/> See Certifications
+                    </a>
+                </t>
+                <t t-else="">
+                    <div class="card p-3 mb-3" t-foreach="certifications" t-as="certification">
+                        <div class="row">
+                            <div class="col-sm-3" t-esc="certification['record'].create_date" t-options="{'widget': 'datetime'}"/>
+                            <div class="col-sm-4">
+                                <t t-if="certification['record'].slide_id">
+                                    <a t-attf-href="/slides_survey/slide/get_certification_url?slide_id=#{certification['record'].slide_id.id}"
+                                        t-esc="certification['record'].survey_id.title"/>
+                                </t>
+                                <t t-else="">
```

Comment:
You can use:

```
<a t-if="blabla">
<span t-else="">
```

It reduces indentation need when introducing a lot of "t" elements.